content/power-on-failure-laptop.md

```diff
@@ -20,7 +20,7 @@ tableOfContents: true
 3. Confirm the outlet you are connecting to works by plugging in another device.
 4. Plug in the AC Adapter to the wall and computer. Laptops have a charging indicator in front with a light. Please provide the light color(s) and icon underneath it to the support team.
      - 4A. If the battery is removable, unplug the machine, remove the battery, and plug the machine back into the power source without the battery installed. Please provide the light status to the support team.
-5. To perform a hardware reset, unplug the system. Disconnect the CMOS battery for 30 seconds. Reconnect the CMOS battery and turn the computer on with the battery installed and the AC Adapter plugged in. Instructions to disconnect the CMOS battery can be found for your model [here](https://tech-docs.system76.com/README.html).
+5. To perform a hardware reset, unplug the system. Disconnect the main battery and CMOS battery for 30 seconds. Reconnect both the main battery and CMOS battery and turn the computer on with the battery installed and the AC Adapter plugged in. Instructions to disconnect the CMOS battery can be found for your model [here](https://tech-docs.system76.com/README.html).
```

Comment:
>Reconnect both the main battery and CMOS battery and turn the computer on with the battery installed and the AC Adapter plugged in.

Still feels clumsy to me. How about something like:

>Reconnect the CMOS battery and the main battery, connect the AC adapter, and power the computer on.

It's more of a direct list of steps and doesn't mention the main battery twice, but feel free to tweak it a bit.